### PR TITLE
Use source for drupal/core for faster composer installs

### PR DESCRIPTION
**Pull request by jonpugh**

### CODE REVIEW

**Comment by ElijahLynn** [comment]
W00t!!!! This is going to shave a minute or two of 3 failed attempts to download Drupal Core by composer through the network TIC/firewall!

Nice work @jonpugh!

**Comment by jonpugh** [comment]
Ok, so this PR has no changes that should affect any tests.

Looks like there are problems with master branch.

**Comment by jonpugh** [comment]
> PHP Fatal error:  Trait 'Drupal\Tests\taxonomy\Traits\TaxonomyTestTrait' not found in /var/aegir/projects/cms/pr917/docroot/vendor/weitzman/drupal-test-traits/src/Entity/TaxonomyCreationTrait.php on line 13

@ElijahLynn this is consistent

**Comment by ElijahLynn** [comment]
> > PHP Fatal error:  Trait 'Drupal\Tests\taxonomy\Traits\TaxonomyTestTrait' not found in /var/aegir/projects/cms/pr917/docroot/vendor/weitzman/drupal-test-traits/src/Entity/TaxonomyCreationTrait.php on line 13
> 
> @ElijahLynn this is consistent

Not getting this locally.

**Comment by ElijahLynn** [comment]
Fixed 

> > > PHP Fatal error:  Trait 'Drupal\Tests\taxonomy\Traits\TaxonomyTestTrait' not found in /var/aegir/projects/cms/pr917/docroot/vendor/weitzman/drupal-test-traits/src/Entity/TaxonomyCreationTrait.php on line 13
> > 
> > 
> > @ElijahLynn this is consistent
> 
> Not getting this locally.

But it appears #919 fixed it!

**Comment by ElijahLynn** [comment]
>  Test Failed: `va/tests/phpunit`
> phpunit --exclude-group disabled tests/phpunit --colors=always
> ```
> PHP Fatal error:  Trait 'Drupal\Tests\taxonomy\Traits\TaxonomyTestTrait' not found in /var/aegir/projects/cms/pr917/docroot/vendor/weitzman/drupal-test-traits/src/Entity/TaxonomyCreationTrait.php on line 13
> ```
> 
> * **On:** devshop.cms.va.gov
> * **In:** 0.21s

**Comment by ElijahLynn** [comment]
Were these changes intended?

image

image

image

**Comment by ElijahLynn** [comment]
drupal-test-traits.git is still updated in the lock file, which should not be updated and is causing this failure. We just need to keep the composer.json change I think. 

This PR will speed up CI and our BRD Builds by a couple minutes.

File: composer.json [hunk 1]
```diff
@@ -319,7 +322,7 @@
             "curl -v --proxy socks5h://127.0.0.1:2001 sentry.vetsgov-internal"
         ],
         "va:proxy:sshuttle": [
-            "sudo -E sshuttle -r dsva@52.222.32.121 -e 'ssh -A ' 10.0.0.0/8"
+            "sudo sshuttle -r dsva@52.222.32.121 -e \"ssh -A -i /$HOME/.ssh/id_rsa\" 10.0.0.0/8"
```
Comment (swirtSJW) [inline composer.json]: @ElijahLynn  Do we still need this addition?   Or can we close this PR?

**Comment by ElijahLynn** [comment]
Fixed in #1162, although it doesn't appear to be honoring it.

**Pull request closed without merging**